libs/blocks/marquee-anchors/marquee-anchors.js

```diff
@@ -64,12 +64,14 @@ export default function init(el) {
       const content = i.querySelector(':scope > div');
       const hrefPathEqual = (aTag.href.split(/\?|#/)[0] === window.location.href.split(/\?|#/)[0]);
       const hrefUrl = (hrefPathEqual)
-        ? `${aTag.textContent}`
+        ? `${aTag.textContent.toLowerCase()}`
         : `${aTag.href}`;
       const link = createTag('a', {
         class: 'anchor-link',
         href: hrefUrl,
       }, content);
+      if (aTag.hasAttribute('target')) link.target = aTag.target;
+      if (aTag.hasAttribute('daa-ll')) link.setAttribute('daa-ll', aTag.getAttribute('daa-ll'));
```

Comment:
@JasonHowellSlavin I think it could go either way. I see what you mean and am open to changing this. 

I guess my initial thought was it seemed easier to wrap all the row content into a new <a> element, steal the referenced attributes and remove the tag. The alternative is getting all row children, sans the aTag, and appending them to the aTag element content. Kinda 6's, but your approach might have more warrant w/ less create/remove element. I'll look into it and see if that way works w/ out too much fuss.

Reply:
@JasonHowellSlavin && @overmyheadandbody  - Thanks for the input on this. I have refactored this as mentioned and its looking good. This should be GTG, but if you get a chance take a look.